package.json

```diff
@@ -62,6 +63,7 @@
     "@types/showdown": "~2.0.0",
     "@types/simple-peer": "~9.11.1",
     "handlebars": "4.7.7",
+    "patch-package": "^6.4.7",
```

Comment:
Should be a devDependency

Reply:
Nope. We need to ship it with the types so that users get the patch too, unless you expect no one to touch TinyMCE, then I'd question the point of even shipping the TinyMCE types.

Reply:
Ah, I'm wrong, big surprise. Does not work for downstream users, as it's not run anyway. https://github.com/ds300/patch-package/issues/198#issuecomment-579386635

Looks like a PITA to fix TinyMCE. I'll swap to a devDep as a rebase, but then we need to make the call if it's a worth while for possible impact of downstream users who have `skipLibCheck` set to false.

Reply:
Ah, yeah, I hadn’t thought about the exact implications for downstream users. But to be honest, I don’t really think we should force this on them, and just stick with TS 4.7 for now, unless there is something we _really_ need from TS 4.8.

And if we stick with TS 4.7, we don’t really need this, right?

Reply:
If we stick with TS4.7 we don't directly need this, though I have a few things that internally we'd benefit from.

That said, any downstream user that uses TS4.8 with the `skipLibCheck` set to false (which is *not* the default, btw) would have issues, no matter what we do. Including the patch for making TinyMCE work under TS4.8 means that downstream users can use the included patch and install `patch-package` to patch it if they need it at least.